### PR TITLE
Display more information in finalized newdle

### LINKED ISSUE #191: Add the ability to display all the data even if the newdle is in final state
Once a final date is selected all other information disappears from the view. In most cases, it is still valuable to be able to go back and review the previous newdle state.

### PR DESCRIPTION
**Pull request by panagiotappl**
closes #191 
image

### CODE REVIEW

File: newdle/client/src/components/ParticipantTable.js [hunk 1]
```diff
@@ -102,32 +109,45 @@ function AvailabilityRow({
...
-export default function ParticipantTable({finalDate, setFinalDate}) {
+export default function ParticipantTable({finalDate, setFinalDate, finalized, children}) {
   const availabilityData = useSelector(getParticipantAvailability);
 
   if (availabilityData.length === 0) {
     return null;
   }
 
+  if (finalized) {
+    availabilityData.some(
```
Comment (ThiefMaster) [inline newdle/client/src/components/ParticipantTable.js:131]: `some()` with side effects is a bit strange... I think creating a new array with the correct order would be better.

Comment (panagiotappl) [reply newdle/client/src/components/ParticipantTable.js:131]: Then we would have to use conditions to know which array to use (whether it's sorted version or not)

Comment (ThiefMaster) [reply newdle/client/src/components/ParticipantTable.js:131]: You can check inside `getParticipantAvailability` if a slot is the final one using the `final_dt` from the `getNewdle` selector and then sort the list accordingly

File: newdle/client/src/components/summary/SummaryPage.js [hunk 2]
```diff
@@ -61,27 +61,37 @@ export default function SummaryPage() {
...
-            )}
-            <Button className={styles['create-event-button']}>Create event</Button>
+          <div className={styles.container}>
+            <Header className={styles.header} as="h2">
+              {newdle.title} will take place on:
+            </Header>
+            <ParticipantTable
+              finalDate={newdle.final_dt}
+              setFinalDate={setFinalDate}
+              finalized={true}
```
Comment (ThiefMaster) [inline newdle/client/src/components/summary/SummaryPage.js:71]:
```suggestion
              finalized
```

File: newdle/client/src/selectors.js [hunk 3]
```diff
@@ -98,15 +98,23 @@ export const newdleHasParticipantsWithoutEmail = createSelector(
   participants => participants.some(p => p.email === null)
 );
 export const getParticipantAvailability = createSelector(
+  getNewdle,
   getNewdleTimeslots,
   getNewdleParticipants,
-  (timeslots, participants) => {
-    return timeslots.map(timeslot => {
```
Comment (ThiefMaster) [inline newdle/client/src/selectors.js:104]: Wouldn't `_.sortBy(..., x => x.startDt !== final_dt);` around the original code do the job as well?

File: newdle/client/src/selectors.js [hunk 4]
```diff
@@ -98,15 +98,19 @@ export const newdleHasParticipantsWithoutEmail = createSelector(
...
-      const available = participants
-        .filter(part => ['available', 'ifneedbe'].includes(part.answers[timeslot]))
-        .map(part => ({...part, fullyAvailable: part.answers[timeslot] === 'available'}));
-      const unavailable = participants.filter(part => part.answers[timeslot] === 'unavailable');
-      return {startDt: timeslot, available, unavailableCount: unavailable.length};
-    });
+  ({final_dt}, timeslots, participants) => {
+    return _.sortBy(
```
Comment (ThiefMaster) [inline newdle/client/src/selectors.js:105]: You could have gone back to the braces-less version of the arrow function ;)

**Comment by ThiefMaster** [comment]
There are some eslint warnings:

```
./src/components/summary/SummaryPage.js
  Line 12:8:  'FinalDate' is defined but never used  no-unused-vars

./src/components/ParticipantTable.js
  Line 4:9:  'Button' is defined but never used  no-unused-vars
```

**Pull request merged**